index.js

```diff
@@ -1,12 +1,9 @@
 "use strict";
 
 function uuid() {
```

Comment:
```suggestion
function uuid() {
  // https://stackoverflow.com/a/2117523/2601552
```

Reply:
Let's include the link where you pulled this solution from as a comment in the code.

Reply:
I found also a problem with the gulp minifying when trying to compile framebus and not handling well the arrow function:
[23:17:29] framebus.min.js 50.1 kB
[23:17:29] 'build' errored after 2.18 s
[23:17:29] GulpUglifyError: unable to minify JavaScript
Caused by: SyntaxError: Unexpected token: operator «>»
File: /Users/jhgascon/Code/framebus/framebus.min.js
Line: 5
Col: 59

Issue is solved with standard function(c) on the other hand. 
Correction submitted.